src/key_value/key_value_client.h

```diff
@@ -11,29 +11,29 @@
 #include <vector>
 
 #include "key_value.grpc.pb.h"
-
+#include "key_value_interface.h"
 
 namespace csci499 {
 using grpc::Channel;
 
 using kvstore::KeyValueStore;
 
 // key value client implementtaion for csci499
-class KeyValueClient {
+class KeyValueClient : public KeyValueInterface {
  public:
   explicit KeyValueClient(std::shared_ptr<Channel> channel)
       : stub_(KeyValueStore::NewStub(channel)) {}
 
   virtual ~KeyValueClient() {}
 
   // package and send rpc call put to server
-  bool Put(const std::string& key, const std::string& value);
+  virtual void Put(const std::string& key, const std::string& value);
```

Comment:
If this implements the interface exactly, you don't need method comments for any method that overrides the parent for whom the parent's method comment is descriptive enough.